Assets/Scripts/Actors/FootstepSound.cs

```diff
@@ -1,9 +1,13 @@
 ﻿using UnityEngine;
 using System.Collections;
 
-public class FootstepSound : MonoBehaviour {
+public class FootstepSound : MonoBehaviour 
+{
+	[SerializeField] AudioSource[] footstepSounds;
 
-	[SerializeField] void PlayerWalkSound(){
-		SoundManager.instance.PlaySoundAtPosition ("Footstep", transform.position-transform.up);
+	[SerializeField] void PlayerWalkSound()
+	{
+		SoundManager.instance.Play3DSoundAtPosition ( footstepSounds[Random.Range(0, footstepSounds.Length)],
+		                                              transform.position - transform.up );
```

Comment:
We should probably have the transform root be at the players feet then. If the player is standing at (0, 0, 0), they should be standing on top of the x-z plane, not chest-deep in the middle of it.